.github/workflows/ci-latest-release.yml

```diff
@@ -113,13 +113,17 @@ jobs:
         with:
           ref: main
 
-      - name: Pull a specific version of KubeArmor from Docker Hub
+      - name: Install regctl
+        uses: iarekylew00t/regctl-installer@v1
```

Comment:
This workflow is also [installing cosign](https://github.com/IAreKyleW00t/regctl-installer/blob/main/action.yml#L29) under the hood, so we can also use `cosign copy` command with `sigstore/cosign-installer`. 
This will also copy the signatures, SBOM and attestations by default. 
Having said that, `regctl` works very smooth, too if we include `--digest-tags`.